fastlane/Helpers/Keychain.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2022 Nimble. All rights reserved.
 //
 
+import Foundation
+	
 enum Keychain {
 
     static func create() {
```

Comment:
Did we miss this change? 🤔 
```
if isCi() {
     // Create new custom self-hosted keychain
} else {
	// Use default keychain
}
```

Reply:
As we add a step to clean up the keychain, we no longer need this 👍